### ISSUE TITLE
Unmet peer dependency for react on npm 3.7.1 and node 5.5.0

### ISSUE THREAD

**Issue by robmadole** [jurassix/react-immutable-render-mixin#17]
I think the peer dependency for react may need a version.

With npm 3.7.1 I can start with this package.json:

```
{
  "name": "t",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": { },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "immutable": "^3.7.6",
    "react": "^0.14.7"
  }
}
```

And run `npm install --save react-immutable-render-mixin`

It results in:
```
t@1.0.0 /Users/robmadole/Development/src/FortAwesome/fortawesome/t
├── UNMET PEER DEPENDENCY react@0.14.7
└── react-immutable-render-mixin@0.9.4 

npm WARN react-immutable-render-mixin@0.9.4 requires a peer of react@ but none was installed.
npm WARN t@1.0.0 No description
npm WARN t@1.0.0 No repository field.
```

**Comment by jurassix** [comment]
Thx for the report. I'll push a patch shortly.

**Comment by zerkms** [comment]
It does not look like a correct behaviour for the npm (a bug?). It's not reproducible on npm 3.3.12 and the `peerDependencies` look as necessary.

**Comment by zerkms** [comment]
Wait a second, I could reproduce it on npm v3.6.0. That is odd

**Comment by jurassix** [comment]
@zerkms Can you test with "*" and see if the issue goes away?

**Comment by zerkms** [comment]
@jurassix I'm trying that (+ `>=0.0.0`, while both must be identical as per the grammar anyway) atm

**Comment by zerkms** [comment]
@jurassix It indeed does solve it

```
$ npm install --save zerkms/react-immutable-render-mixin#ISSUE-17
t@1.0.0 /home/vagrant/src/immutable-mixin
└── react-immutable-render-mixin@0.9.4  (git://github.com/zerkms/react-immutable-render-mixin.git#fc07161637454672c7dd72e9081bee1d43322a2f)
```

**Comment by jurassix** [comment]
@zerkms fixed in v0.9.5

**Comment by robmadole** [comment]
@jurassix @zerkms thank you!
